src/components/NumberInput/NumberInput.tsx

```diff
@@ -48,7 +49,7 @@ function NumberInput({
       type="text"
       inputMode="decimal"
       placeholder={placeholder}
-      className={className}
+      className={cx(className, "text-white placeholder:text-slate-100")}
```

Comment:
Nice, maybe if the color of placehholders is the same throughout the site, we set it globally in `tailwind.css` ?

Reply:
No, colors are differ for NumberInput and SearchInput for instance